### PR TITLE
feat(content): Umbral Reach Update

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

the reach

## Summary
The "Umbral Reach" as it is currently in the game, is the area of systems with a dark haze to the right of Gegno space.

### "New" Missions

Currently, the only mission regarding the UR is the Rulei ship acknowledging you visited it. I've decided to expand upon this mission a bit more and have some text show up as you enter the first time, and give a little bit more to the Rulei conversation, both invoking even more of a sense of "desolate emptiness" in the narration. As a result, two new short logs are now added, and the actual label for the area is added via event. (I think it looks great with the shrouded mechanic since the systems disappear when you travel away from them - see below.)

Unless I go in and add conditions on the `to offer` to include whether or not you've done these missions prior to the PR, players will experience these missions again even if they're only slightly changed. Regardless, they do have logs. Either I can let it replay as-is, or have a patch condition that gives players the logs/event based on having previously done the missions, and then block these "new" ones from offering.

Something else to consider: there are Jje spawns prior to the `on enter` npc one that is there explicitly to describe them. They're intentionally supposed to be found in the entire area, but with very rare spawns. This is more to guarantee one spawns for the player here, but it is a small chance the player seems them prior to this system. I could add fleets after the mission via events, but I feel like this is an edge case that we can ignore so we don't have to add more to a save file.

spooky

### Added Features and Adjustments

Since the addition of the UR to the game, we've added some mechanics that I feel help add on to the mysterious nature of the region along with the above. Each system has been given both "shrouded", and a "starfield density".

Using the shrouded mechanic, the area is now more mysteriously hidden, and as you navigate, the surrounding systems will disappear once more. After talking with @Azure3141, this is a more subtle use of "shrouded" versus the Avgi PR's "Tangled Shroud", as this still only has one way in-out, meanwhile the TS is much more of a maze. Also, presumably, from a new player perspective, it is more likely they experience the TS before this area.

The starfield density lets the area be even more "spooky" as it appears less lit up by stars, and further plays into the shadiness of the hazes. In addition, I've edited the asteroids, mineables, spawns, and stellar objects of the region - mostly removed a lot of them - to continue invoking that dreadful emptiness. It's not just supposed to be an uninhabited area - it's supposed to be eerily uninhabited.

## Save File
Will add one if necessary, otherwise, JD back over to spookyville.

## Artwork Checklist
(If any artwork was added or changed by this PR, the following must be provided.)
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
~~- [ ] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}~~
~~- [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~~
 
 Labels don't have 2x's nor do we typically have assets for them. They're essentially glorified text. However, I have elected to put them in the copyright, since there is one in there for The Graveyard.

### CODE REVIEW

**Comment by Saugia** [comment]
I'd consider that option too, since the Rulei generally speak with less words. The very first iterations of the name were "(dark word) + reach", to imply that the darkness is reaching out, but I wouldn't be necessarily opposed to dropping the reach part.

**Comment by TheGiraffe3** [comment]
I think changing the name of the Umbral Reach would make it sound less mysterious.
Is there a reason that we can't change the name of the system in human space?

File: data/rulei/rulei.txt [hunk 1]
```diff
@@ -90,8 +90,30 @@ mission "Rulei Planet"
...
 
-mission "Umbral Reach"
+	on enter "L-6181"
+		fail
+		conversation
+			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, rather, it appears as some sort of "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
+
+	npc
+		government "Indigenous Lifeform (Acheron)"
+		personality timid unconstrained uninterested mute
```
Comment (TheGiraffe3) [inline data/rulei/rulei.txt:112]:
```suggestion
		personality mute timid unconstrained uninterested
```

File: data/rulei/rulei.txt [hunk 2]
```diff
@@ -100,13 +122,21 @@ mission "Umbral Reach"
 		has "First Contact: Rulei: offered"
 		has "visited system: L-118"
 	on enter "Kanguwa"
+		event "antumbral reach label"
+		log "Factions" "Rulei" `The Rulei seem to acknowledge the void of systems east of the core as the "Antumbral Reach", and appear to have some sort of emotional connection to the region. Other than stating to "beware" the area, they will not further comment on it.`
```
Comment (TheGiraffe3) [inline data/rulei/rulei.txt:126]:
```suggestion
		log "Factions" "Rulei" `The Rulei seem to acknowledge the void of systems east of the core as the "Antumbral Reach," and appear to have some sort of emotional connection to the region. Other than stating to "beware" the area, they will not further comment on it.`
```

**Review by TheGiraffe3 — COMMENTED** [review]
Really liking the addition of `shrouded` though. It makes the space a lot more strange, and also it feels more hostile.
There is _nothing_ out here.

File: data/rulei/rulei.txt [hunk 3]
```diff
@@ -90,8 +90,30 @@ mission "Rulei Planet"
...
+	on enter "L-118"
+		log `Discovered a region of emptiness east of the core, where systems are shrouded by a vast, eerie darkness. Other than dim stars and a few planets, nothing exists.`
+		conversation
+			`Upon arriving into this system, you notice a strange murkiness curtaining the stars beyond, and your scanners are having trouble making sense of it. The few stellar objects present are eerily blotted out by a vast, nebulous shade. Aside from them, it's as if there is nothing here. Even the shade itself is nothing more than the emptiness of space itself, and throughout it, no one. Nothing.`
 
-mission "Umbral Reach"
+	on enter "L-6181"
+		fail
+		conversation
+			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, rather, it appears as some sort of "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```
Comment (Daeridanii1) [inline data/rulei/rulei.txt:108]: I like the flavor of this conversation, but there's some internal inconsistency. 
If the player's scanners don't read the Jje as a lifeform, how do they know to describe it as a "creature"? Does it approach the ship such that FL can see it with their own eyes, and thus confirm its living nature? i.e.
```suggestion
			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you catch a glimpse of what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, rather, it appears as some sort of "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```
A "'collection' of matter" is equally nebulous: presumably the sensors describe _anything with mass_ as a collection of matter. If it's their inability to describe it that you want to emphasize, maybe something like the following would better convey that?
```suggestion
			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```

**Review by Daeridanii1 — COMMENTED** [review]
Very neat! The (Ant)Umbral Reach is a great place for the `shrouded` and `starfield density` attributes to be used.

I have to say I'm less fond of "Antumbral Reach" than I am of "Umbral Reach." It adds more complexity to the name, but not much more meaning (outside the most technically-minded reading), so I read it as somewhat less "punchy" and memorable than the original.

**Comment by tekker2234** [comment]
I agree that the Umbral Reach sounds a lot more punchy and memorable than Antumbral Reach. I'd also say that the possible confusion about what the umbral reach is (and the possible association with Umbral in Human space) is cleared up pretty well with the addition of the label.

File: data/rulei/rulei.txt (hunk 3 above)
Comment (Saugia) [reply data/rulei/rulei.txt:108]: The latter sounds good. As for it being "a creature", I wouldn't put it past the same kind of recognition the player has for something like void sprites, or presumably the Vyrmeid. It's still "appears to be", as it's a moving form of sorts amongst dead systems, so I don't think it's too far fetched.

**Comment by Saugia** [comment]
After some discussion, comments, etc. with players and devs, I've decided to keep the original name. I still think it's the best as well.

File: data/rulei/rulei.txt [hunk 4]
```diff
@@ -100,13 +122,21 @@ mission "Umbral Reach"
 		has "First Contact: Rulei: offered"
 		has "visited system: L-118"
 	on enter "Kanguwa"
+		event "umbral reach label"
+		log "Factions" "Rulei" `The Rulei seem to acknowledge the void of systems east of the core as the "Umbral Reach", and appear to have some sort of emotional connection to the region. Other than stating to "beware" the area, they will not further comment on it.`
 		conversation
 			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
 			`	"Beware..."`
-			`	"...Umbral Reach."`
+			`	"...Umbral Reach.."`
```
Comment (TheGiraffe3) [inline data/rulei/rulei.txt:130]:
```suggestion
			`	"...Umbral Reach..."`
```

**Review by TheGiraffe3 — CHANGES_REQUESTED** [review]

File: data/rulei/rulei.txt (hunk 2 above)
Comment (TheGiraffe3) [reply data/rulei/rulei.txt]: For some reason this didn't get resolved?

**Review by TheGiraffe3 — APPROVED** [review]
Other than https://github.com/endless-sky/endless-sky/pull/10298#discussion_r1666179191.

File: data/rulei/rulei.txt (hunk 2 above)
Comment (roadrunner56) [reply data/rulei/rulei.txt]: The original line was right, the comma is supposed to be outside of the quotation.

File: data/rulei/rulei.txt [hunk 5]
```diff
@@ -90,8 +90,30 @@ mission "Rulei Planet"
...
+	on enter "L-118"
+		log `Discovered a region of emptiness east of the core, where systems are shrouded by a vast, eerie darkness. Other than dim stars and a few planets, nothing exists.`
+		conversation
+			`Upon arriving into this system, you notice a strange murkiness curtaining the stars beyond, and your scanners are having trouble making sense of it. The few stellar objects present are eerily blotted out by a vast, nebulous shade. Aside from them, it's as if there is nothing here. Even the shade itself is nothing more than the emptiness of space itself, and throughout it, no one. Nothing.`
 
-mission "Umbral Reach"
+	on enter "L-6181"
+		fail
+		conversation
+			`As you continue through this space of emptiness, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```
Comment (roadrunner56) [inline data/rulei/rulei.txt:108]:
```suggestion
			`As you continue through this empty space, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```

File: data/rulei/rulei.txt [hunk 6]
```diff
@@ -90,8 +90,30 @@ mission "Rulei Planet"
...
+	on enter "L-118"
+		log `Discovered a region of emptiness east of the core, where systems are shrouded by a vast, eerie darkness. Other than dim stars and a few planets, nothing exists.`
+		conversation
+			`Upon arriving into this system, you notice a strange murkiness curtaining the stars beyond, and your scanners are having trouble making sense of it. The few stellar objects present are eerily blotted out by a vast, nebulous shade. Aside from them, it's as if there is nothing here. Even the shade itself is nothing more than the emptiness of space, and throughout it, no one. Nothing.`
 
-mission "Umbral Reach"
+	on enter "L-6181"
+		fail
+		conversation
+			`As you continue through this empty space, you come across nothing but planets of silence and diminished stars. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```
Comment (roadrunner56) [inline data/rulei/rulei.txt:108]: I personally think this sounds better, but up to you.
```suggestion
			`As you continue through this empty space, you come across nothing but diminished stars and planets of silence. However, upon entering this system, you are met with what appears to be a creature of some kind. Strangely, your scanners don't read it as any sort of lifeform, categorizing it only as a mere "collection" of matter. It blends with space itself, as if it were simply a reflection of the darkness beyond.`
```

**Comment by roadrunner56** [comment]
Tested and it seems like it works fine.

**Comment by VaelophisNyx** [comment]
Yeah shrouded is just confusing and frustrating to deal with in general
unvisited system node graphics are far too hard to see without straining to do so for this to be left as is imo
And yeah no, the ship computer's job is to remember these places, so unless there's some concrete reason given to why it won't, it just feels like slapping the player with a random debuff with no explanation at all

**Pull request merged**